packages/SwingSet/src/kernel/kernel.js

```diff
@@ -150,9 +150,10 @@ export default function buildKernel(
     return kernelSlog.vatConsole(vatID, origConsole);
   }
 
-  // runQueue entries are {type, vatID, more..}. 'more' depends on type:
-  // * deliver: target, msg
-  // * notify: kernelPromiseID
+  // runQueue and acceptanceQueue entries are {type, more..}. 'more' depends on type:
```

Comment:
This might be a good place for a note on which queue is which.

```
// message flow: vat -> syscall -> acceptanceQueue -> runQueue -> delivery -> vat
```